From bed0ff13bf09b3c6190f9ff0cf6f7182a96a0faf Mon Sep 17 00:00:00 2001
From: Simon Howard <fraggle@soulsphere.org>
Date: Wed, 10 Jun 2015 22:38:19 -0400
Subject: [PATCH] setup: Fix help URL for level warp menu.

The help URL for the warp menu was linking to the multiplayer start
game menu.
---
 src/setup/multiplayer.c | 13 +++++++++++--
 1 file changed, 11 insertions(+), 2 deletions(-)

diff --git a/src/setup/multiplayer.c b/src/setup/multiplayer.c
index a36f4da..3fe6a82 100644
--- a/src/setup/multiplayer.c
+++ b/src/setup/multiplayer.c
@@ -37,6 +37,7 @@
 #define MULTI_START_HELP_URL "http://www.chocolate-doom.org/setup-multi-start"
 #define MULTI_JOIN_HELP_URL "http://www.chocolate-doom.org/setup-multi-join"
 #define MULTI_CONFIG_HELP_URL "http://www.chocolate-doom.org/setup-multi-config"
+#define LEVEL_WARP_HELP_URL "http://www.chocolate-doom.org/setup-level-warp"
 
 #define NUM_WADS 10
 #define NUM_EXTRA_PARAMS 10
@@ -715,9 +716,17 @@ static void StartGameMenu(char *window_title, int multiplayer)
     txt_widget_t *iwad_selector;
 
     window = TXT_NewWindow(window_title);
-    TXT_SetWindowHelpURL(window, MULTI_START_HELP_URL);
 
-    TXT_AddWidgets(window, 
+    if (multiplayer)
+    {
+        TXT_SetWindowHelpURL(window, MULTI_START_HELP_URL);
+    }
+    else
+    {
+        TXT_SetWindowHelpURL(window, LEVEL_WARP_HELP_URL);
+    }
+
+    TXT_AddWidgets(window,
                    gameopt_table = TXT_NewTable(2),
                    TXT_NewSeparator("Monster options"),
                    TXT_NewInvertedCheckBox("Monsters enabled", &nomonsters),
-- 
2.4.3

